### DISCUSSION TITLE
Lock shell keyboards togethers in deepin-terminal

### DISCUSSION THREAD

**Discussion by rmannibucau** [linuxdeepin/developer-center#9573] [Features Request & Ideas | 特性请求 & 头脑风暴]
A feature I really like from terminator is the ability to split a screen in multiple terms then affect to each sub-shell-window an identifier and all sub-windows with the same identifier will type the same thing.

This is commonly used to ssh on N machines and execute the same serie of commands on all of them.

Can be neat to get it in deepin-terminal.

**Comment by BLumia** [comment]
cc @meiyixiang
